clang-tools-extra/clangd/ClangdServer.cpp

```diff
@@ -112,7 +112,10 @@ struct UpdateIndexCallbacks : public ParsingCallbacks {
                  // Index outlives TUScheduler (declared first)
                  FIndex(FIndex),
                  // shared_ptr extends lifetime
-                 Stdlib(Stdlib)]() mutable {
+                 Stdlib(Stdlib),
+                 // We have some FS implementations that rely on infomration in
+                 // the context.
+                 Ctx(Context::current().clone())]() mutable {
```

Comment:
@sam-mccall, @usx95 could you please vet this change? 
I'm on vacation since tomorrow and I won't have time to go through this today.
It's probably correct, I'm just missing context to know for sure and getting it will take time.

Reply:
oops that shouldn't be here, commit from a different branch messed things up. (FWIW, it's part of https://github.com/llvm/llvm-project/pull/87611)